workflows/ingest/masters.go

```diff
@@ -11,17 +16,14 @@ import (
 	"github.com/bcc-code/bccm-flows/services/vidispine/vscommon"
 	"github.com/bcc-code/bccm-flows/utils/wfutils"
 	"go.temporal.io/sdk/workflow"
-	"path/filepath"
-	"regexp"
-	"strconv"
-	"strings"
 )
 
 type MasterParams struct {
 	Metadata *ingest.Metadata
 
-	OrderForm OrderForm
-	Directory paths.Path
+	OrderForm  OrderForm
+	Directory  paths.Path
+	SourceFile paths.Path
```

Comment:
Make this a pointer/nilable instead of checking path.Path, as JSON will throw errors when this is default.